packages/airnode-validator/templates/0.3/config.json

```diff
@@ -61,6 +61,52 @@
           "__message": "__fullPath: Currently only \"evm\" is supported"
         }
       },
+      "chainOptions": {
```

Comment:
This is not correct. You should not update the `0.3` templates, but `0.4`.
This means you need to copy all of the templates in `0.3` first and then update them in the new `0.4` directory.

Reply:
I recommend doing this in a separate PR (since there will be many changes because of the copying)

Reply:
Okay, thanks, I'll do this now before any other actions.